clicache/integration-test/AttributesFactoryTestsN.cs

```diff
@@ -24,20 +24,35 @@ namespace Apache.Geode.Client.UnitTests
 {
   using NUnit.Framework;
   using Apache.Geode.DUnitFramework;
-  using Apache.Geode.Client;
+  
+ 
 
   [TestFixture]
   [Category("group1")]
   [Category("unicast_only")]
   [Category("generics")]
   public class AttributesFactoryTests : UnitTests
   {
+
+    private UnitProcess m_client1, m_client2;
+
+    protected override ClientBase[] GetClients()
```

Comment:
Why did this test require adding clients? As far as I can tell all the testing runs local to the test runner and are not invoked in remote clients.